importScripts/sampleRecords/additionalAdministration.destination.json

```diff
@@ -10,7 +10,7 @@
       "ca_objects.is_deaccessioned":"0",
       "ca_objects.LastEditDate":"30 July 2015",
       "ca_objects.LastEditBy":"PENDS",
-      "ca_storage_locations":"Artworks Store Room C2"
+      "usualLocationHierarchy":" (id:1, ) => Artworks Store Room C2 (id:2, room)"
```

Comment:
This is a hierarchy now. The root has id:1 and no type. The rest of the values have a type.